qubesbuilder/plugins/chroot_rpm/mock/fedora.cfg

```diff
@@ -17,6 +17,10 @@ config_opts['use_bootstrap'] = False
 config_opts['plugin_conf']['bind_mount_enable'] = os.environ.get("BIND_MOUNT_ENABLE", False)
 config_opts['plugin_conf']['bind_mount_opts']['dirs'].append(('@BUILDER_DIR@/plugins', '/plugins' ))
 
+# mount the pesign socket into the chroot
+config_opts['plugin_conf']['bind_mount_opts']['dirs'].append(('/var/run/qubes-pesign', '/var/run/qubes-pesign' ))
+config_opts['nspawn_args'] += ['--bind=/var/run/qubes-pesign']
```

Comment:
Aren't those two meaning the same thing? Why both are needed?

Reply:
I've copied paste some snippet in a thread but I've tested and only the second one is needed.